src/mme-app/utils/mmeStatesUtils.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020-present Infosys Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <utils/mmeStatesUtils.h>
+
+using namespace mme;
+using namespace SM;
+
+/***************************************
+* MmeStatesUtils : on_state_entry
+***************************************/
+ActStatus MmeStatesUtils::on_state_entry(ControlBlock& cb)
+{
+    // TODO: Start state guard timer if configured
```

Comment:
Could you please elaborate what is Guard timer ?  Example would really help.

Reply:
A timer is started whenever a state is set and stopped when the state is changed. On expiry of the state, cleanup actions can be undertaken by handling the state_guard_timeout event for each state.
This helps to prevent the procedure being stuck in a state for too long due to unexpected conditions.
Updated the documentation.